lib/config/openssh/openssh.go

```diff
@@ -52,11 +52,16 @@ Host *.{{ $clusterName }} {{ $dot.ProxyHost }}
 # Flags for all {{ $clusterName }} hosts except the proxy
 Host *.{{ $clusterName }} !{{ $dot.ProxyHost }}
     Port {{ $dot.Port }}
-    {{- if eq $dot.AppName "tsh" }}
+{{- if eq $dot.AppName "tsh" }}
     ProxyCommand "{{ $dot.ExecutablePath }}" proxy ssh --cluster={{ $clusterName }} --proxy={{ $dot.ProxyHost }}:{{ $dot.ProxyPort }} %r@%h:%p
-{{- end }}{{- if eq $dot.AppName "tbot" }}
+{{- end }}
+{{- if eq $dot.AppName "tbot" }}
+{{- if $dot.PureTBotProxyCommand }}
+    ProxyCommand "{{ $dot.ExecutablePath }}" ssh-proxy-command --destination-dir={{ $dot.DestinationDir }} --proxy-server={{ $dot.ProxyHost }}:{{ $dot.ProxyPort }} --cluster={{ $clusterName }} --tls-routing={{ $dot.TLSRouting }} --connection-upgrade={{ $dot.ConnectionUpgrade }} --resume={{ $dot.Resume }} --user=%r --host=%h --port=%p
```

Comment:
Maybe a weird issue since this is generating broken configs on my machine. It generates this `ProxyCommand`:

```
ProxyCommand "/home/tim/projects/teleport-b/build/tbot" -d ssh-proxy-command --destination-dir=/home/tim/projects/teleport-b/tbot-user/id --proxy-server=teleport.ethernet.fyi:443 --cluster=teleport.ethernet.fyi --tls-routing=true --connection-upgrade=false --resume=true --user=%r --host=%h --port=%p
```

This fails with the extremely helpful error message:
```
$ "/home/tim/projects/teleport-b/build/tbot" -d ssh-proxy-command --destination-dir=/home/tim/projects/teleport-b/tbot-user/id --proxy-server=teleport.ethernet.fyi:443 --cluster=teleport.ethernet.fyi --tls-routing --no-connection-upgrade --resume=true --user=tim --host=teleport.teleport.ethernet.fyi --port=3022
tbot: error: unexpected true
```

After a bit of debugging, this apparently is referring to the CLI args. tbot's boolean flags do not accept `--foo=true` (?) and instead must be `--foo` or `--no-foo`. So this apparently should be the valid ProxyCommand?

```
"/home/tim/projects/teleport-b/build/tbot" ssh-proxy-command --destination-dir=/home/tim/projects/teleport-b/tbot-user/id --proxy-server=teleport.ethernet.fyi:443 --cluster=teleport.ethernet.fyi --tls-routing --no-connection-upgrade --resume --user=tim --host=teleport.teleport.ethernet.fyi --port=3022
```

Specifically: `--tls-routing --no-connection-upgrade --resume`

Reply:
Hah - I figured I might run into this. I was going to manual test yesterday but ran out of time so thanks for catching, I'll fix this morning.

Reply:
[cdce31c](https://github.com/gravitational/teleport/pull/41482/commits/cdce31c397ff21a96781521327ba4bf7ec70e28b)